pkg/types/auth.go

```diff
@@ -18,6 +18,13 @@ type AWSAuth struct {
 	SessionToken    string `bson:"session_token" json:"session_token"`
 }
 
+type AzureAuth struct {
+	ClientID       string `bson:"client_id" json:"client_id"`
+	ClientSecret   string `bson:"client_secret" json:"client_secret"`
+	TenantID       string `bson:"tenant_id" json:"tenant_id"`
+	SubscriptionID string `bson:"subscription_id" json:"subscription_id"`
```

Comment:
Do you really need the `bson` tags? I went through much of the code and it might just be a red herring.

Reply:
@patrickdappollonio please stop reviewing code until marked ready or requested. I've asked you this before and I will make a formal HR complaint if you continue doing this without being asked.